assets/css/_vehicle_properties_card.scss

```diff
@@ -103,5 +102,6 @@
 .c-vehicle-work-info th {
   padding-right: 36px;
   text-align: start;
+  vertical-align: top;
```

Comment:
I'm wondering if it would be worth updating `vehicle-work-info` to use the `PropertiesList` component for consistency with VPP, or if these styles should continue to further diverge as they are here

Reply:
Personally, I don't like the interface that `PropertiesList` provides, but I think making them use the same component is a good idea.

Maybe there's a way to keep the existing interface for `PropertiesList` and also integrate the interface that `TrNameValue` provides?

Reply:
It seems like the child component`PropertyRow` of `PropertyList` could be updated to use `TrNameValue` with only a small modification to add a `className` to `TrNameValue`. I'm going to make a separate ticket to do so since I don't want to increase the scope of this ticket to include possible styling regressions for existing components using the `PropertiesList` component.

Reply:
Added a ticket: https://app.asana.com/0/1148853526253420/1204762479194649/f